libraries/apollo-ast/src/jvmTest/kotlin/com/apollographql/apollo/graphql/ast/test/SDLWriterTest.kt

```diff
@@ -18,15 +19,6 @@ class SDLWriterTest {
     }
   }
 
-  @Test
-  fun typeRedefinitionInspectionIsIgnored() {
-    val sdlSchema = File("${CWD}/test-fixtures/sdl/type_redefinitions.graphqls")
-
-    checkExpected(sdlSchema) {
-      it.parseAsGQLDocument().getOrThrow().withBuiltinDefinitions().toSDL("    ")
-    }
-  }
```

Comment:
I'm not 100% sure what this was testing. Removed it. Might add new ones later on.